src/app/homepage/homepage.component.scss

```diff
@@ -0,0 +1,85 @@
+@use '@angular/material' as mat;
+@use '@angular/material-experimental' as matx;
+
+@mixin theme($theme) {
+  .recent-container {
+    height: 40vh;
+    padding-left:32px;
+    padding-right: 32px;
+    font: mat.get-theme-typography($theme, headline-large);
+  }
+
+  .user-sets-container{
+    height: 46vh;
+    padding-left:32px;
+    padding-right: 32px;
+  }
+  .recent-title{
+    font: mat.get-theme-typography($theme, headline-medium);
+    font-weight: bold;
+    line-height: 0em;
+  }
+  .recent-carousel{
+    overflow-x: scroll;
+  }
+  .recent-carousel-items{
+    margin-top: 4px;
+    list-style-type: none;
+    height:252px;
+    width:max-content;
+  }
+
+  /* Placeholder Visuals */
+  .study-set-card{
+    background-color: mat.get-theme-color($theme, surface-container-lowest);
+    float: left;
+    font: mat.get-theme-typography($theme, body-medium);
+    margin-right: 1em;
+    width: 255px;
+    height: 247px;
+    padding: 12px;
+    box-shadow: 0 3px 6px rgba(0,0,0,0.16), 0 3px 6px rgba(0,0,0,0.23);
+  }
+  .study-set-description{
+    padding-bottom:36px;
+  }
+  .preview-grid{
+    display: grid;
+    grid-template-columns: auto auto;
+    grid-gap: 4px;
+  }
+  .preview-card{
+    background-color: mat.get-theme-color($theme, primary-container);
+    font: mat.get-theme-typography($theme, body-large);
+    padding: 20px;
+    text-align: center;
+  }
+
+  .user-sets-title{
+    font: mat.get-theme-typography($theme, headline-medium);
+    font-weight: bold;
+    line-height: 0em;
+  }
+
+  .user-sets-list{
+    height: 85%;
+    overflow-y: scroll;
+    padding-top: 4px;
+  }
+  .user-sets-card{
+    background-color: mat.get-theme-color($theme, surface-container-lowest);
+    font: mat.get-theme-typography($theme, headline-small);
+    width:99.5%;
+    height:67px;
+    box-shadow: 0 3px 6px rgba(0,0,0,0.16), 0 3px 6px rgba(0,0,0,0.23);
+    line-height: 1em;
+    padding-left: 16px;
+    text-align:left;
+    vertical-align: center;
+    margin-bottom: 12px;
+    margin-left: 4px;
+
+  }
```

Comment:
Add this to the new component for user study set list item. You can pick how to name the new component.